### ISSUE TITLE
Amplify pull cannot sync override custom attribute cloudformation template to local

### ISSUE THREAD

**Issue by Nutchawat** [aws-amplify/amplify-cli#9098]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

16.13.0

### Amplify CLI Version

7.3.2

### What operating system are you using?

Mac

### Amplify Categories

auth

### Amplify Commands

pull, push, Not applicable

### Describe the bug

To add a custom attribute to the cognito generated by amplify, I use the command ```amplify override auth```.
I added a custom attribute ```tenant_id``` to the ```override.ts``` file.
Amplify generates a cloudformation template with the override custom attribute and successfully deploys it to the remote after using the ```amplify push``` command.
After that, sync cloudformation template from remote to local using the ```amplify pull``` command.
The issue is that the override custom attribute cloudformation template is missing.

```
modified:   amplify/backend/auth/<poolname>/build/<poolname>-cloudformation-template.json

    "Schema": [
        {
            "Mutable": true,
            "Name": "email",
            "Required": true
-       },
-       {
-           "AttributeDataType": "String",
-           "Mutable": true,
-           "Name": "tenant_id",
-           "Required": false
        }
    ],
```

### Expected behavior

The override custom attribute cloudformation template can be sync to local by "amplify pull" command.

### Reproduction steps

1. amplify override auth (add custom attribute)
2. amplify push (push overrided template from local to remote)
3. amplify pull (fails: remote cannot pull template with overrided to local)

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Additional information

git status when amplify pull is done.
modified:   amplify/backend/amplify-meta.json
modified:   amplify/backend/auth/<poolname>/build/<poolname>-cloudformation-template.json
deleted:    amplify/backend/package-lock.json
deleted:    amplify/backend/package.json
deleted:    amplify/backend/tsconfig.json

**Comment by Straubulous** [comment]
This is resolved in latest release (7.5.4)

**Comment by akshbhu** [comment]
Hi 

Here is the merged PR: https://github.com/aws-amplify/amplify-cli/pull/9078
This is fixed in our latest release  : 7.5.4

you can try : ```npm install -g @aws-amplify/cli@latest```

Let me know if this doest work  ,ill reopen this issue again
